src/main/java/monitoringsystemturbo/presenter/MainPresenter.java

```diff
@@ -106,7 +113,24 @@ public void onNextDay() {
     }
 
     @FXML
-    public void onAddApplication() {
+    public void onAddApplication() throws IOException, ClassNotFoundException {
+        addApplicationController.showAddView();
+        Application application=addApplicationController.getNewApplication();
+        if (application != null) {
+            application.createFileIfNeeded();
+            loadedApplications.add(application);
+
+            applicationList.setItems(FXCollections.observableList(loadedApplications));
```

Comment:
to samo co wyżej, brak inicjalizacji zmiennej loadedApplications

Reply:
naprawione , gdy dodalem brakujący commit